extension/user/getRegistrationUrl.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @param {SDKContext} context
+ * @param {Object} input
+ * @param {function} cb
+ */
+module.exports = function (context, input, cb) {
+  const url = ''
```

Comment:
The frontend needs a way to decide, if the registration is supposed to be done via external website or via a native view. This pipeline might be reused for that. We should talk in the team about what the best way would be.

Reply:
Just add a topic to our Retros